source/mainnet/smart-contracts/tutorials/piggy-bank/deploying.rst

```diff
@@ -209,4 +211,32 @@ You can find additional information about updating a smart contract instance in
 .. image:: ./images/pb_tutorial_21.png
    :width: 100 %
 
+.. note::
+   You can find the meaning of common error codes in this `list <https://docs.rs/concordium-std/3.0.0/concordium_std/#signalling-errors>`_.
+   This error was thrown by the ``concordium-std`` crate. The error codes start from ``i32::MIN`` and go upwards.
+   ``-2147483636`` corresponds to ``NotPayableError``. In contrast, error types of the smart contract start from ``-1`` and go downwards.
+
+Smart contract errors
+---------------------
+
+.. note::
+
+   Ensure that the ``--sender`` flag uses the <account-name>
+   that initialized the piggy bank smart contract instance (the owner of this smart contract instance).
```

Comment:
Please mention that it will result in the error shown below. Similar to what you do in the first error section.